packer/build_image.sh

```diff
@@ -36,19 +34,13 @@ print_usage() {
     echo "  --ec2-instance-type     Set EC2 instance type to use while building the AMI. If empty will use defaults per architecture"
     exit 1
 }
-LOCALDEB=0
 DOWNLOAD_ONLY=0
```

Comment:
`DOWNLOAD_ONLY` - can be removed as well

Reply:
removed